config/locales/hyrax.en.yml

```diff
@@ -1,10 +1,10 @@
 ---
 en:
   hyrax:
-    product_name:           "Hyku"
+    product_name:           "Hyku Commons"
     product_twitter_handle: "@HydraInABox"
     institution_name:       "Institution"
-    institution_name_full:  "Institution"
+    institution_name_full:  "The Institution Name"
```

Comment:
Change this back to "Institution" for this and all other languages

Reply:
Institution has been changed in en, de